src/api-client.ts

```diff
@@ -147,6 +147,79 @@ export class RevAiApiClient {
             payload.getHeaders(), 'json', payload, TWO_GIGABYTES);
     }
 
+    /**
+     * See https://www.rev.ai/docs#section/Human-Transcription-(Labs)
+     * Submit media given a URL for transcription by a human. The audio data is downloaded from the URL.
+     * @param mediaUrl Web location of media to be downloaded and transcribed
+     * @param options (optional) Options submitted with the job, see RevAiHumanTcJobOptions object
+     * @returns Details of the submitted job
+     */
+    async submitHumanTranscriptionJobUrl(
+        mediaUrl: string,
+        options?: RevAiHumanTranscriptionJobOptions
+    ): Promise<RevAiApiJob> {
+        options = this.filterNullOptions({
+            transcriber: 'human',
+            media_url: mediaUrl,
+            ...(options || {})
```

Comment:
what if they also specify transcriber in these options? If we're going to have separate methods for this then `transcriber` should go into the `RevAiJobOptions` subclass

Reply:
I understand the motivation for creating a separate transcription function for human, but I have the same worry as Kyle that this introduces additional complications.

One of the selling points of having human and machine in the same transcription api is ease of implementation and switch. I believe that changing a single option `transcriber` is a simpler change than to replace the name of a function.

If you were writing conditionals which set the transcriber you would have to write more if you had to use another function:

```
if (audioQuality < 0.7) {
    submitHumanTranscriptionJobUrl(...)
} else {
    submitJobUrl(...)
}
```

In the other way:

```
submitJobUrl(..., {...options, transcriber: audioQuality < 0.7 ? 'human' : 'machine_v2' );
```